src/lib/isCaseRecordable.ts

```diff
@@ -4,7 +4,7 @@ const isCaseRecordable = (annotatedHearingOutcome: AnnotatedHearingOutcome): boo
   const isPncQuery = annotatedHearingOutcome.PncQuery !== undefined
   const isThereARecordableOffence =
     annotatedHearingOutcome.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.Offence.some(
-      (offence) => offence.RecordableOnPNCindicator
+      (offence) => offence.RecordableOnPNCindicator === "Y"
```

Comment:
This was previously set to a boolean in zod however in legacy bichard `validator.xml` this is an Enum that comes from data `[yes-no.json](https://github.com/ministryofjustice/bichard7-next-data/blob/main/output-data/data/yes-no.json)`